src/main/java/nextstep/subway/path/controller/PathController.java

```diff
@@ -0,0 +1,23 @@
+package nextstep.subway.path.controller;
+
+import nextstep.subway.path.dto.PathRequest;
+import nextstep.subway.path.dto.PathResponse;
+import nextstep.subway.path.service.PathService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequestMapping("/paths")
+@RestController
+public class PathController {
+    private final PathService pathService;
+
+    public PathController(PathService pathService) {
+        this.pathService = pathService;
+    }
+
+    @GetMapping
+    public PathResponse findPaths(PathRequest request) {
+        return pathService.findPaths(request);
```

Comment:
아래처럼 `ResponseEntity`를 반환하는게 좋겠네요.

```suggestion
    public ResponseEntity<PathResponse> findPaths(PathRequest request) {
        return ResponseEntity.ok(pathService.findPaths(request));
```